tools/creality_printer_test_server.py

```
@@ -190,6 +200,132 @@ def _gen():
+                        continue
+                    packet.pts = i * step
+                    packet.dts = packet.pts
+                    packet.time_base = self._time_base
+                    self._packets.append(packet)
+            finally:
+                container.close()
+        finally:
+            try:
+                os.unlink(path)
+            except OSError:
+                pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

(SIM105)